### PR TITLE
update config to use 2.0v of plugin

### PR DESCRIPTION
**Pull request by dnwarnock**

### CODE REVIEW

File: async-websocket-wlpcfg/pom.xml [hunk 1]
```diff
@@ -86,118 +68,33 @@
...
-                            <goal>verify</goal>
-                        </goals>
-                    </execution>
-                </executions>
                 <configuration>
                     <summaryFile>${project.build.directory}/test-reports/fvt/failsafe-summary.xml</summaryFile>
                     <reportsDirectory>${project.build.directory}/test-reports/fvt</reportsDirectory>
+                    <systemPropertyVariables>
+                    	<liberty.test.port>${testServerHttpPort}</liberty.test.port>
+                    	<war.name>${warName}-${warVersion}</war.name>
```
Comment (cthigh) [inline async-websocket-wlpcfg/pom.xml]: How is this `war.name` property used?  Before we added `<stripVersion>true</stripVersion>`, the war name had the version but we took it off to match what was configured in server.xml file.

**Comment by cthigh** [comment]
At some point this project could be built with Gradle or Maven, but the build.gradle file was removed about a year ago. The documentation, https://github.com/dnwarnock/sample.async.websockets/blob/master/docs/Using-cmd-line.md, still talks about building with Gradle. We should update the doc.

File: async-websocket-wlpcfg/pom.xml [hunk 2]
```diff
@@ -86,118 +68,33 @@
                         <default.http.port>${testServerHttpPort}</default.http.port>
                         <default.https.port>${testServerHttpsPort}</default.https.port>
                     </bootstrapProperties>
-                    <install />
+                    <stripVersion>true</stripVersion>
+                    <install/>
```
Comment (cthigh) [inline async-websocket-wlpcfg/pom.xml]: Don't forget you were going to add a `version` parameter to this.  Did that work?

**Comment by cthigh** [comment]
Also, remember we saw that the help said to download liberty which didn't make sense.  We can fix that and close https://github.com/WASdev/sample.async.websockets/issues/27.

**Review by cthigh — CHANGES_REQUESTED** [review]
See my various comments on the files changed and more general comments.

File: async-websocket-wlpcfg/pom.xml (hunk 2 above)
Comment (dnwarnock) [reply async-websocket-wlpcfg/pom.xml]: Worked fine!

**Comment by dnwarnock** [comment]
Should I just remove the details about gradle and downloading liberty then?

**Comment by patricktiu** [comment]
I was hoping we can merge this multi-module project into a single war project. See the PR for sample.async.servletnio, https://github.com/josephca/sample.async.servletnio/pull/1.

**Comment by dnwarnock** [comment]
@patricktiu our thoughts were that this sample could act as an example of using multiple modules to construct a project

**Comment by cthigh** [comment]
Downloading info should be removed.  I wish we knew why they removed the gradle build.  Maybe the project was converted to a multi-module project and then gradle didn't work?  @patricktiu do you not see any value for showing a multi-module project? I was thinking it was good to show for someone who wanted to work that way.

**Comment by cthigh** [comment]
@patricktiu and I had a chance to talk about this sample.  We will leave this one as a multi-project. Go ahead and update the doc to remove the references to building with gradle.

**Comment by dnwarnock** [comment]
@cthigh Requested doc changes have been made

**Review by cthigh — APPROVED** [review]

**Pull request merged**